controllers/endPoints.controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchEndPoints } = require('../models/endPoints.model');
+
+exports.getEndPoints = (request, response, next) => {
+    fetchEndPoints()
+    .then(endpoint => {
```

Comment:
How many endpoints are we expecting to see here? Just one, or multiple? The variable name should match what's being represented.